media/stylus/components/compat-tables/bc-legend.styl

```diff
@@ -0,0 +1,28 @@
+/*
+legend
+====================================================================== */
+
+.bc-legend {
+    -moz-column-count: 2;
+    -webkit-column-count: 2;
```

Comment:
CSS3 Multiple column layout not supported by: Firefox (37,38), Chrome (42,43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.3)